tensorflow/python/ops/array_ops.py

```diff
@@ -3532,7 +3546,76 @@ def _get_paddings_constant(paddings):
     return [_get_paddings_constant(x) for x in paddings]
   else:
     return paddings
+  
+def _wrap_pad(tensor, paddings, name=None):
+  """Helper to create wrap pad option with ops already built.
 
+  See `pad` for more info on how this function works.
+  """
+  # use constant padding to create a tensor with final shape and original values.
```

Comment:
nit: for comments, start with a capital letter and end with punctuation (here and all comments below).  They should be full sentences.

Reply:
👍